src/sql/src/catalog.rs

```diff
@@ -485,6 +485,10 @@ pub trait CatalogSchema {
 pub struct RoleAttributes {
     /// Indicates whether the role has inheritance of privileges.
     pub inherit: bool,
+    /// The password for the role
+    pub password: Option<String>,
```

Comment:
Is this the plaintext password or a hashed version? Can you also update the comment to mention this separate from the Frontegg/external auth used in the cloud product